src/daemon/abrtd.c

```diff
@@ -174,19 +174,13 @@ static void queue_post_create_process(struct abrt_server_proc *proc)
 
     char *worst_dir = NULL;
     const double max_size = 1024 * 1024 * abrt_g_settings_nMaxCrashReportsSize;
-    while (libreport_get_dirsize_find_largest_dir(abrt_g_settings_dump_location, &worst_dir, ignored) >= max_size
+    while (libreport_get_dirsize_find_largest_dir(abrt_g_settings_dump_location, &worst_dir, ignored, proc->dirname) >= max_size
```

Comment:
Actually, what's the difference between the new argument and the one preceding it? Shouldn't that fulfill the same function? I may be missing something.

Reply:
That's what I thought too, but we want to ignore both the dir that was just created by the process that triggered the directory trimming and any directory from which a backtrace is being generated.